examples/tokens/readme.md

```diff
@@ -0,0 +1,85 @@
+# Token Transfer Example
+
+This is an example of using the tupelo-wasm-sdk to establish, mint, and send tokens from one ChainTree to another.
+
+The example program will use the public Tupelo testnet, so you will need to be connected to the Internet.
+
+There's a video walkthrough of token transfer here:
+
+[![Play video](http://img.youtube.com/vi/kjysUm0eTKc/1.jpg)](https://youtu.be/kjysUm0eTKc "Wallet Walkthrough")
+
+Unlike other systems, Tupelo models individual objects *first* (as ChainTrees) and then layers tokens on top of those individual objects.
+
+ChainTrees may mint their own tokens and send and receive tokens from other ChainTrees. Within a ChainTree a token can have an arbitrary name (unique to the individual ChainTree), the canonical name of the token is the DID of the ChainTree with the arbitrary name appended. For example, if ChainTree `did:tupelo:xyz` minted a `stockToken` the canonical token name would be `did:tupelo:xyz:stockToken`
+
+There are four transactions in the Tupelo API that deal with tokens: `ESTABLISHTOKEN`, `MINTTOKEN`, `SENDTOKEN`, and `RECEIVETOKEN`. `ESTABLISHTOKEN` defines the token specification and monetary policy, `MINTTOKEN` creates new tokens once a token has already been established, and `SENDTOKEN`/`RECEIVETOKEN` token transfers already minted tokens between ChainTrees.
```

Comment:
I'm wondering if it's better to say that `MINTTOKEN` mints new tokens, instead of "creates". Might help disambiguate vs. establishing tokens?

Also, should it be "`SENDTOKEN`/`RECEIVETOKEN` transfers already minted tokens[...]"? I.e., looks like a stray "token" after "`SENDTOKEN`/`RECEIVETOKEN`.

Reply:
I think using 'mint' is using the same word to define the word. Minting is "creating" tokens and "establishing" is just defining. good catch on the stray 'token'

Reply:
fixorated stray token.